projects/angular/src/accordion/accordion-panel.spec.ts

```diff
@@ -176,6 +181,13 @@ describe('ClrAccordionPanel', () => {
         accordionPanel.togglePanel();
         expect(component.change).toHaveBeenCalledWith(false);
       });
+
+      it('when [clrAccordionPanelHeadingEnabled] is not set at all, heading role should not be present', () => {
```

Comment:
This test does not belong in the "Template API -> Output (clrAccordionPanelOpenChange)" test block. It should be in the "View" test block with other test.